public/client/search.js

```diff
@@ -29,4 +36,35 @@ function request(bookId, pointCost, title){
             alert('Error occured while retrieving book')
         }
     })
+}
+
+function bookDetails(isbn, pointCost, userPoints){
+        $.ajax({
+            url: '/googleapi/isbn/' + isbn,
+            method: 'GET',
+            dataType: 'json',
+            success: function (res) {
+                console.log(res);
+                
+                var details = document.getElementById('details');
```

Comment:
I think this line can be deleted as it doesn't appear to be used anywhere. Line 49